scripts/zf_mkdist

```diff
@@ -186,7 +196,12 @@ if [ -z "$version" ]; then
     version=$(git rev-parse HEAD)
 fi
 
-[ $tarball_count -eq 1 ] || usage
+[ $tarball_count -eq 1 ] || [ -d /usr/include/ci ] || usage
+
+if $shim && [ -z "$onload_tarball" ]; then
```

Comment:
Not sure if this file is already shellchecked, but is shellcheck fine with `$shim` by itself?

Reply:
Doesn't complain on this line. Raised #33 for the existing issues